examples/internal/proto/examplepb/unannotated_echo_service.swagger.yaml

```diff
@@ -12,33 +12,28 @@ openapiOptions:
           license:
             name: BSD 3-Clause License
             url: https://github.com/grpc-ecosystem/grpc-gateway/blob/master/LICENSE.txt
-          version: '1.0'
+          version: "1.0"
           extensions:
             x-something-something: yadda
         schemes:
-        - HTTP
-        - HTTPS
-        - WSS
+          - HTTP
+          - HTTPS
+          - WSS
         consumes:
-        - application/json
-        - application/x-foo-mime
+          - application/json
+          - application/x-foo-mime
         produces:
-        - application/json
-        - application/x-foo-mime
+          - application/json
+          - application/x-foo-mime
         responses:
-          '403':
+          "403":
             description: Returned when the user does not have permission to access the resource.
-          '404':
+          "404":
             description: Returned when the resource does not exist.
             schema:
               jsonSchema:
                 type:
-                - STRING
-          '418':
-            description: I'm a teapot.
-            schema:
-              jsonSchema:
-                ref: ".grpc.gateway.examples.internal.proto.examplepb.NumericEnum"
```

Comment:
I had to remove this since it relies on this file being compiled together with `a_bit_of_everything.proto`, which seems wrong anyway.